wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -15,16 +15,180 @@
 
 <main id="main" class="site-main">
 
+<?php
+if ( '' === get_query_var( 'search' ) && empty( $_GET ) && is_post_type_archive() ) :
+	?>
+	<section>
+		<div class="row align-middle between section-heading section-heading--with-space large-space">
+			<h1 class="section-heading_title bigger"><?php esc_html_e( 'Lesson Plans', 'wporg-learn' );?></h1>
+			<a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) . '?_view=all' ); ?>" class="button button-large button-secondary">
+				<?php esc_html_e( 'Browse all lesson plans', 'wporg-learn' ); ?>
+			</a>
+		</div>
+
+		<hr>
+
+		<div class="row align-middle between section-plan_description">
+			<div class="section-plan_text">
```

Comment:
Semantically this is best as a paragraph

```suggestion
			<p class="section-plan_text">
```